gateway-modules/apim-ballerina-configs/configs/org/wso2/carbon/apimgt/gateway/throttle/Throttling.bal

```diff
@@ -73,13 +73,13 @@ function isRequestThrottled( message msg) (boolean){
     string apiVersion = keyValidationDto.apiVersion;
     string applicationId = keyValidationDto.applicationId;
 
-    if (authorizedUser == ""){
-        http:setStatusCode( msg, HTTP_UNAUTHORIZED);
-        messages:setProperty(msg, THROTTLED_ERROR_CODE, BLOCKED_ERROR_CODE);
-        messages:setProperty(msg, THROTTLED_OUT_REASON, THROTTLE_OUT_REASON_REQUEST_BLOCKED);
-        setThrottledResponse(msg);
-        return true;
-    }
+    //if (authorizedUser == ""){
```

Comment:
Shall we remove the commented code?

Reply:
@sabeena1217  we shouldn't comment these lines afaik it affects the normal gateway behaviour